ui/src/message_popup/playtest/smith_experience_page.jsx

```diff
@@ -52,8 +63,8 @@ export default React.createClass({
   // Making questions from the cohort
   onStart(email) {
     const {cohortKey} = this.state;
-    const allQuestions = smithScenario.questionsFor(cohortKey);
-
+    const {facilitated} = this.props;
+    const allQuestions = smithScenario.questionsFor(cohortKey, (facilitated));
```

Comment:
Passing facilitated here is good, and you're reading it in on the other end correctly too.  In general, passing boolean values can be hard to understand, and for options like this it's usually more explicit to pass it as an object that says what the boolean value is.

So here you could change this to `{isFacilitated: facilitated}` and then update the two places in the scenario code that check for this so that they take `options` and then they check for `options.isFacilitated`.